File: lógica/JS/estrutura-para/tabuada.js
// Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10.

let numero, tabuada;
numero = parseInt(prompt('Digite o valor da tabuada'));

for(let i = 1; i < 11; i++) {
    console.log(`${i} x ${numero} = ${i * numero}`);
}

File: lógica/Python/estrutura-repetitiva/tabuada.py
# Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10

n = int(input('Digite um número: '))

for i in range(1, 11):
    tabuada = n * i
    print(f'{n} x {i} = {tabuada}')